src/utils/graphql/rsk/generated.tsx

```diff
@@ -8713,6 +8713,35 @@ export type BorrowFieldsFragment = {
   transaction: { __typename?: 'Transaction'; id: string };
 };
 
+export type GetVestedHistoryQueryVariables = Exact<{
+  user?: InputMaybe<Scalars['String']>;
+  skip: Scalars['Int'];
+  pageSize: Scalars['Int'];
+}>;
+
+export type GetVestedHistoryQuery = {
+  __typename?: 'Query';
+  stakeHistoryItems: Array<{
+    __typename?: 'StakeHistoryItem';
+    id: string;
+    action: StakeHistoryAction;
+    amount?: string | null;
+    timestamp: number;
+    lockedUntil?: number | null;
+    transaction: { __typename?: 'Transaction'; id: string };
+  }>;
+};
+
+export type VestedHistoryFieldsFragment = {
```

Comment:
@pietro-maximoff  this should be the VestingHistoryItem not the StakeHistoryItem